src/Uno.UI/Microsoft/UI/Xaml/Controls/InfoBar/InfoBarPanel.Properties.cs

```diff
@@ -1,40 +1,82 @@
-﻿// MUX reference InfoBarPanel.properties.cpp, commit 533c6b1
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+// MUX reference InfoBarPanel.properties.cpp, commit 533c6b1
 
 using Windows.UI.Xaml;
 
-namespace Microsoft.UI.Xaml.Controls
+namespace Microsoft.UI.Xaml.Controls.Primitives
```

Comment:
I might have missed something in the recent symbol matching in the sync generator. It should get caught from there.

Reply:
@Youssef1313 Should I create an issue for this one?

Reply:
Maybe. Not a priority though.